blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarter.java

```diff
@@ -295,8 +303,10 @@ public void run() {
                     TimeUnit.MINUTES.toMillis(savePeriodMins));
         }
 
-        // has the side-effect of causing static initialization of Metrics, starting instrumentation reporting.
-        new RestartGauge(Metrics.getRegistry(), RollupService.class);
+        if (shouldInstantiateRestartGauge) {
```

Comment:
This is still getting really hairy. Trying to mock the RestartGauge constructor with PowerMock requires changing the junit version, and I'm still getting `java.lang.NoClassDefFoundError` and `"java.lang.LinkageError: loader constraint violation`. I vote we leave the flag in place, and work on the `RestartGauge` in a separate PR.